docs/index.rst

```diff
@@ -1,49 +1,135 @@
+   new_to_edx/what_you_can_do
+   new_to_edx/mission
+   new_to_edx/what_our_systems_do
+   new_to_edx/data_model
+   new_to_edx/messaging_model
+   new_to_edx/what_to_read_next
 
+Information for all Open edX developers
+=======================================
 
-Architecture and Best Practices
--------------------------------
+You will likely use some or all of the information in this section as you progress 
+in your Open edX development work. Take a look through the topics here before you begin
+writing code. 
 
-.. list-table::
+.. From the outline: these topics appear to be of common interest. Are they? i18n, XSS, MFEs (Micro-frontends), IDAs (Independently Deployed Applications, a.k.a. Microservices), Django Configuration/Toggles, Named Releases]
 
-   * - Open edX Technology Radar (coming...)
-     - A bird's eye view of the decisions, tools and technologies that have been adopted or rejected for Open edX
-       development.
+.. list-table:: 
+   :widths: auto
 
+   * - :doc:`all_development/roadmap/index`
+     - Getting your device set up for development, testing, and deployment.
+   * - :doc:`all_development/common_tech/index`
+     - Common elements shared by core platform and extension work.
+   * - :doc:`all_development/dev_principles/index` 
+     - Guidelines and standards for all Open edX developers (for example, accessibility and code style).
+   * - :doc:`all_development/contributing/index` 
+     - If you want to contribute to Open edX code or documentation, review these topics.
    * - `Open edX Proposals <https://open-edx-proposals.readthedocs.io/en/latest/>`_
-     - Technical decisions made by the Open edX community, in the form of best practices, architecture design,
-       or development processes.
+     - Help us decide what comes next and see how we made decisions: our technical proposals.
+
+
+.. toctree::
+   :maxdepth: 4
+   :hidden:
+   :caption: Information for all Open edX developers
+
+   all_development/roadmap/index
+   all_development/common_tech/index
+   all_development/dev_principles/index
+   all_development/contributing/index
+
+
+Working with the core Open edX platform
+=======================================
+
+If you want to work with the Open edX core systems (course authoring, need to enumerate), review these topics.
+
+If you want to contribute to the Open edX repositories, see :doc:`all_development/contributing/index`.
+
+.. WHICH TOPICS ARE UNIQUE TO CORE DEVELOPMENT?
+
+* :doc:`working_with_core/working_with_core_topic_1`
+* :doc:`working_with_core/working_with_core_topic_2`
+
+.. toctree::
+   :maxdepth: 4
+   :hidden:
+   :caption: Working with the core Open edX platform
+
+   working_with_core/working_with_core_topic_1
+   working_with_core/working_with_core_topic_2
+
+Extending the Open edX platform
+===============================
+
+If you want to extend Open edX functionality, review these topics.
```

Comment:
"extend functionality" can be ambiguous.  This section is about building new independent components that can be used with Open edX.

Reply:
(At least, the links I read on readthedocs were about that)

Reply:
I expect that the technical content of these files will change significantly over time.